runtime/metrics_types.go

```diff
@@ -111,19 +112,16 @@ func NewMetric(name string, i interface{}) *Metric {
 		t = MetricHistogram
 		m = pools[t].Get().(*Metric)
 		s := v.Snapshot()
-		p := s.Percentiles([]float64{0.5, 0.75, 0.95, 0.99, 0.999, 0.9999})
+		p := s.Percentiles([]float64{0.5, 0.75, 0.95, 0.99, 0.999})
 		m.Measures["count"] = float64(s.Count())
 		m.Measures["max"] = float64(s.Max())
 		m.Measures["mean"] = s.Mean()
 		m.Measures["min"] = float64(s.Min())
-		m.Measures["stddev"] = s.StdDev()
```

Comment:
@raulk I've add calculations for `stddev` and `variance`, but I really don't think we need to be calculating these at runtime and sending them to InfluxDB.

If for some reason someone needs them, we can always enable them. WDYT?

Reply:
I don't think it hurts to mimic the behaviour we're replacing. There's value in calculating these derivatives here -- as calculating them on Influx is would already be doing so with a lesser resolution.

Reply:
99% of charts won't be graphing those. My thinking is that we include them, if someone really needs them or finds them valuable. Basicaly I meant YAGNI :D

Reply:
What's the technical reason to drop them? Are they costly to calculate?

Reply:
One of the main use-cases to include this type is to measure latencies. We care about mean, 95%, 99% and max, when graphing those. We don't want to miss spikes and measurements that happen in interval N to affect measurements in the next interval N+1, hence the resetting.

It is not clear to me why someone would want to graph `stddev` and `variance` for latencies at all - also one is a derivation of the other. Basically we are adding 2 fields that probably won't be used as part of every measurement, sending them across the wire and persisting them - it seems wasteful to me.

Folks are welcome to calculate these off-band with other tools we have build in the SDK, for example the outputs.

---

Why add more data to every measurement, when we don't have an immediate use-case for it?

---

That's my thinking.

---

Bottom line: If someone comes with a use-case that needs those two - it is trivial to add them later... I don't see a reason to add them as part of the SDK today.